ecs-windows-cdk-dotnet/README.md

```diff
@@ -0,0 +1,64 @@
+# Running Windows Containers with Amazon ECS
+
+This AWS CDK application demonstrates how to deploy a application on Windows based Amazon Elastic Container Service (Amazon ECS) Cluster.
+
+This pattern uses .NET as a programming language to create the entire AWS CDK stack. For demonstration purpose, Windows sample image is used (mcr.microsoft.com/windows/servercore/iis), however same steps can be performed if Windows Docker image need to be deployed from Amazon Elastic Container Registry (Amazon ECR) repository.
+
+## Architecture 
+![architecture diagram](images/architecture.png)
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/ecs-windows-cdk-dotnet
+
+**Important**: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [.NET 6](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) installed
+* [AWS Cloud Development Kit](https://docs.aws.amazon.com/cdk/latest/guide/cli.html) (AWS CDK) installed
+
+## Deployment Instructions
+
+1. Clone the project to your local working directory.
+    ```
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change the working directory to this pattern's directory.
+    ```
+    cd ecs-windows-cdk-dotnet/cdk/src
```

Comment:
For me to build work this needs to happen inside the WindowsECS folder.

Reply:
@mavi888 I found an issue with my solution file, now I have fixed and pushed the changes, it should work without going inside WindowsECS folder.